### PR TITLE
PBM-1042 Added option to define custom path for mongod

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by janwieremjewicz — APPROVED** [review]

File: docs/usage/restore.md [hunk 1]
```diff
@@ -124,6 +140,26 @@ During the restore, Percona Backup for MongoDB restores the data on the node whe
 
 Alternatively, you can place the encryption key to all nodes of the replica set. Then the restore is successful and complete on all nodes. This approach is faster and may suit for large data sets (terabytes of data). However, we recommend to rotate the encryption keys afterwards. Note, that key rotation is not available after the restore [for data-at-rest encryption with HashiCorp Vault key server](https://docs.percona.com/percona-server-for-mongodb/latest/vault.html#vault). In this case, consider using the scenario with partially done restore. 
 
+### Define `mongod` binary location
+
+!!! admonition "Version added: 2.0.4"
+
+During physical restores, Percona Backup for MongoDB performs several internal restarts of the database. By default, it uses the location of the `mongod` binaries from the `$PATH` variable to access the database. If you have defined the custom path to the `mongod` binaries, make Percona Backup for MongoDB aware of it by specifying this path in the configuration file:
```
Comment (sandraromanchenko) [inline docs/usage/restore.md:147]: remove "internal"

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/reference/restore-options.md [hunk 2]
```diff
@@ -18,4 +22,16 @@ The number of documents to buffer.
...
 
-The number of workers that add the documents to buffer.
\ No newline at end of file
+The number of workers that add the documents to buffer.
+
+### mongodLocation
+
+*Type*: string
+
+The custom path to `mongod` binaries. When undefined, Percona Backup for MongoDB uses the default path to make internal database restarts during physical restore.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/reference/restore-options.md:31]: Fixed

File: docs/reference/restore-options.md [hunk 3]
```diff
@@ -18,4 +22,16 @@ The number of documents to buffer.
...
+
+### mongodLocationMap
+
+*Type*: array of strings
+
+The list of custom paths to `mongod` binaries on every node. Percona Backup for MongoDB uses the vales to make internal restarts of the database during physical restore.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/reference/restore-options.md:37]: Fixed the typo. Why the default when this is the custom path?

File: docs/usage/restore.md (hunk 1 above)
Comment (nastena1606) [reply docs/usage/restore.md:147]: Done

File: docs/usage/restore.md [hunk 4]
```diff
@@ -65,12 +67,22 @@ The default values were adjusted to fit the setups with the memory allocation of
 
     The MongoDB version for both backup and restore data must be within the same major release.
 
+### Preconditions
+
+As the precondition to a physical restore, shut down all `mongos` nodes to stop clients from accessing the database while restore is in progress. This ensures that the final restored data doesn’t differ from the backed-up data.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/usage/restore.md:72]: Updated

File: docs/reference/restore-options.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sandraromanchenko) [reply docs/reference/restore-options.md]: Sorry, my bad, I've read it the same way as above option. It's ok now.

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**